web/js/codeworld.js

```diff
@@ -221,6 +221,59 @@ function initCodeworld() {
         }
     });
     window.codeworldEditor.refresh();
+    window.codeworldEditor.on("cursorActivity", function() {
+        const prevDiv = document.getElementById("function-details");
+        if (prevDiv) prevDiv.remove();
+
+        const enclosingFunction = getEnclosingFunction(window.codeworldEditor);
+
+        if (!enclosingFunction || !enclosingFunction.string ||  !window.codeWorldSymbols[enclosingFunction.string]) {
+            return;
+        }
+        const cursor = window.codeworldEditor.getCursor();
+        const currentToken = window.codeworldEditor.getTokenAt(cursor);
+        const lineTokens = window.codeworldEditor.getLineTokens(cursor.line);
+        const argIndex =
+            lineTokens.filter(
+                elem =>
+                    getEnclosingFunction(
+                        window.codeworldEditor,
+                        {
+                            line: cursor.line, ch: elem.end
+                        })
+                        .string === enclosingFunction.string
+                    && elem.type && elem.type !== "keyword"
+                    && elem.type!=="bracket-0"
```

Comment:
"bracket-0" is almost surely wrong.  Did you want to look for brackets at the same level as the open-bracket that introduced the function?  (But see the other comment about moving this logic to the editor mode instead; this reinforces that suggestion.)